cmd/kubeadm/app/phases/controlplane/manifests_test.go

```diff
@@ -540,33 +539,6 @@ func TestGetControllerManagerCommand(t *testing.T) {
 		cfg      *kubeadmapi.ClusterConfiguration
 		expected []string
 	}{
-		{
-			name: "custom cluster name for " + cpVersion,
```

Comment:
this test case is primarily for testing the ClusterName field.
we should keep the test case and only remove the cfg.FeatureGates input.